net/nghttp3/Portfile

```diff
@@ -3,26 +3,25 @@
 PortSystem              1.0
 PortGroup               github 1.0
 
-github.setup            ngtcp2 nghttp3 0.13.0 v
+github.setup            ngtcp2 nghttp3 0.14.0 v
 revision                0
```

Comment:
please add the "github.tarball_from" line here with either "releases" (preferred if upstream provides those), otherwise "archive". 

Same comment applies to the other Portfile.

Reply:
Done for both.